support-e2e/tests/consent-management.test.ts

```diff
@@ -1,5 +1,6 @@
 import { expect, test } from '@playwright/test';
 
+/** These have been covered in smoke/cron tests */
```

Comment:
What is the purpose of these comments?

Reply:
So I know they've been recreated and can delete them one the new setup is complete (docs, alerting etc).